docs/docs/concepts/identity-schema.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-id: identity-data-model
-title: Identity Data Model
+id: identity-schema
+title: Identity Schema
```

Comment:
What does the SEO say? :thinking: :fox_face:

Reply:
If I search for `Identity Data Model` our doc is placed 2.
For `Identity Schema` we are in place 3.

Reply:
Is there anything else I didn't consider? 
We probably have to add the old 
`identity-data-models` to https://github.com/ory/web/blob/master/static/_redirects ?

Reply:
I don't know much about SEO, but changing URLs is always not as easy as just changing it.

Reply:
This will need a PR for in here:

https://github.com/ory/web/blob/1fa63437980966acfbfec8ac55a8601ba127962c/static/_redirects#L252